<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimestampDialog</class>
 <widget class="QDialog" name="TimestampDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Anwesenheit-Dialog</string>
  </property>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>dd.MM.yyyy</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDate">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Datum</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditUserName">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMatriculationNumber">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelUserName">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMatriculationNumber">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Matrikelnummer</string>
   </property>
  </widget>
  <widget class="QTableView" name="attendanceTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>401</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
